index.html

```diff
@@ -29,8 +29,8 @@ <h1> Lykke with Weather <i class="fa-solid fa-umbrella-beach"></i> </h1> <br>
              </div>
              <div class="temperature-item">
                 <div class="temperature">9</div>
-                <div class="unit">°C</div>
-                <p class="condition-text">, Overcast Clouds</p>
+                <div class="unit">°C </div>
```

Comment:
if you want to place the unit at the top of the temperature and use `position: relative;` as the solution, you have to place the unit div inside the temperature div!
This is because the unit div is going to get the `position: absolute;` and it indicates its position based on the first parent with `position: relative;`
Here the temperature is not unit's parent but its sibling :)